mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -43,13 +43,13 @@ class DistanceSensorItem {
 	{ }
 
 	// params
-	bool is_subscriber;		//!< this item is a subscriber, else is a publisher
-	bool send_tf;			//!< defines if a transform is sent or not
-	uint8_t sensor_id;		//!< id of the sensor
+	bool is_subscriber;	//!< this item is a subscriber, else is a publisher
+	bool send_tf;		//!< defines if a transform is sent or not
+	uint8_t sensor_id;	//!< id of the sensor
 	double field_of_view;	//!< FOV of the sensor
 	tf::Vector3 position;	//!< sensor position
```

Comment:
OK. But please(!), instead, go to specific lines to change so to orientation handling work ok (that's the purpose of this PR).